test/e2e/data/e2e_conf.yaml

```diff
@@ -198,7 +198,7 @@ intervals:
   default/wait-cluster: ["20m", "10s"]
   default/wait-control-plane: ["30m", "10s"]
   default/wait-worker-nodes: ["30m", "10s"]
-  default/wait-delete-cluster: ["5m", "10s"]
+  default/wait-delete-cluster: ["10m", "10s"]
```

Comment:
Great find! I thought there was some issue with the code making it stuck. Turns out it was just a too tight deadline :sweat_smile: